docs/application/dotnet/tutorials/certificates/permit-device-to-install-apps.md

```diff
@@ -0,0 +1,14 @@
+# Permit Device To Install Applications
+
+In the menu bar, click `Tools > Tizen > Tizen Device Manager`.
+And right-click anywhere in the file explorer of the `Device Manager` and select `Permit to install applications`.
```

Comment:
Global comment:
It is recommended to use **Bold** style for File names, menu names, user control names and so on. Please verify whether the usage of `code tag` is proper.

Please update.

Reply:
done